<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicItemEditionDialog</class>
 <widget class="QDialog" name="BasicItemEditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar item edition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="lineEditTitle"/>
   </item>
   <item row="1" column="0">
    <widget class="QDateEdit" name="dateEditStart"/>
   </item>
   <item row="1" column="1">
    <widget class="QTimeEdit" name="timeEditStart"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelTo">
     <property name="text">
      <string>to</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QTimeEdit" name="timeEditEnd"/>
   </item>
   <item row="1" column="4">
    <widget class="QDateEdit" name="dateEditEnd"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTextEdit" name="textEditDescription"/>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BasicItemEditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BasicItemEditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
